src/client/pages/Debugger/components/QuoteData.tsx

```diff
@@ -377,11 +378,15 @@ export const QuoteData: React.FC<OfferProps> = ({ sessionToken }) => {
                       )}
 
                       <Button
+                        type="submit"
                         background={colorsV3.purple500}
                         foreground={colorsV3.gray900}
-                        type="submit"
+                        disabled={props.isSubmitting}
                       >
                         Create quote
+                        {props.isSubmitting && (
```

Comment:
I would suggest that in addition to having the loading indicator on the button also not render the form when `isSubmitting` (or just as much of the form as is visible when we have an error). Since you can submit the form by selecting an input field and pressing return key it makes sense that a loading indicator would be visible even if you don't scroll to the bottom of the page.